### DISCUSSION TITLE
Is it possible to download chrome profiles that are already ready to go?

### DISCUSSION THREAD

**Discussion by ForceDev1** [devlikeapro/waha#249] [Q&A]
I have chrome profiles already ready to go. Is it possible to upload them?

**Comment by allburov** [comment]
Hi! 
Yes, you can try to do it by putting them under `.session/webjs/{sessionName}/session-{sessionName}/Default` folder, it's "chromium"  profiles under the hood

After that if you start `{sessionName}` by `POST /api/sessions/start` - it'll see there's authentication data and perform all the work

**Reply by ForceDev1** [reply]
And is it possible to move profiles to the filesystem without accessing the docker interface?

**Reply by allburov** [reply]
Yes, if you're using host volume to save the files https://waha.devlike.pro/docs/how-to/sessions/#session-persistent
```
-v `pwd`/.sessions:/app/.sessions
```

You can place files directly in the folder on host system and than run the docker.

Let me know that solution works for you and if it does and you need to have an endpoint for that - we could add, like `PUT /api/{session}/chrome-profile` which would accept tar.gz\zip file and saved that internally in docker

**Reply by ForceDev1** [reply]
Yes. It would be very nice to organize an api for this. How long will it take?

**Reply by allburov** [reply]
Let me know if the above solution works for your first!

**Reply by ForceDev1** [reply]
I'm not really sure what this is for. Let's say I don't have access to the server with docker, I need to upload account sessions there by url, can this be realized?

**Reply by ForceDev1** [reply]
I was able to move the Default folder to my session and everything worked. But I need to do it on the requests

**Comment by allburov** [comment]
I've created a task for that, subscribe to get the update when it's ready!
https://github.com/devlikeapro/whatsapp-http-api/issues/250
